src/ProgressOnderwijsUtils/Pocos/PocoProperties.cs

```diff
@@ -35,6 +35,17 @@ public Type PocoType
         IndexByName = dictionary;
     }
 
+    public bool TryGetByName(string name, [NotNullWhen(true)] out IPocoProperty<T>? property)
+    {
+        if (IndexByName.TryGetValue(name, out var index)) {
+            property = Properties[index];
+            return true;
+        } else {
+            property = null;
+            return false;
+        }
```

Comment:
Maar je bedoelde dit??

```suggestion
            property = IndexByName.TryGetValue(name, out var index)
                ? Properties[index]
                : null;
            return property != null;
```

Reply:
Yes